From 96a7be9637a0d504699b6397fbf718593fcc6f19 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 14 Feb 2018 10:40:05 +0800
Subject: [PATCH 1318/1345] fix: dts: mvebu: a7/8k: correct ap806 gpio bank
 interrupts

commit  ffedfd589353661d4964daddb31775a5a93488cf from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

GIC SPI 22, 23, 24, 25 is connected to MSS_GPIO and AP806 GPIO
interrupt is connected to GIC SPI 30, this patch fixes it.

Change-Id: Id24b702f2c1e36aa713382b2a6d3fcb03e24e83e
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/52311
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index b429d50..65e43b7 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -431,10 +431,7 @@
 				#gpio-cells = <2>;
 				interrupt-controller;
 				#interrupt-cells = <2>;
-				interrupts = <GIC_SPI 22 IRQ_TYPE_LEVEL_HIGH>,
-					<GIC_SPI 23 IRQ_TYPE_LEVEL_HIGH>,
-					<GIC_SPI 24 IRQ_TYPE_LEVEL_HIGH>,
-					<GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
+				interrupts = <GIC_SPI 30 IRQ_TYPE_LEVEL_HIGH>;
 			};
 
 			sd_mmc_reg0: regulator@0 {
-- 
1.7.9.5

